### DISCUSSION TITLE
Websocket connection Issue

### DISCUSSION THREAD

**Discussion by PPnaruto** [rainbow-me/rainbowkit#2149] [Q&A]
Hey, Not able to connect any of the wallet. Not able to see QR code after clicking on the any of the wallet option.
Getting websocket connection issue.

WebSocket connection to 'wss://relay.walletconnect.org/?auth=eyJhbGciOiJFZERTQSIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJkaWQ6a2V5Ono2TWtyd0tiYUM1ZUxmMUhSWnBCVm1CMzhoeHRBNHprZ25DUDl2M1cxV1lzelB1biIsInN1YiI6IjUzNzFhOWUzOTU1NjFlYTk0NWYyOTEwYWExZjJkMTYwOGM3ZDZkYWIwOTBiYTUwNjM3ZWJjYjcyYTQ1ZTZhMzQiLCJhdWQiOiJ3c3M6Ly9yZWxheS53YWxsZXRjb25uZWN0Lm9yZyIsImlhdCI6MTcyNDE0MDc4MiwiZXhwIjoxNzI0MjI3MTgyfQ.ee_HGHvcUC5_1PjsCfD9YfiPid6oFFDym30cXvwg3ZYMEcHf01uDgonN4nwjprHiDZdIZQqh1QA1b3vFGUVXAg&projectId=9fef2fa4b27be5078c4dd997e69fd305&ua=wc-2%2Fjs-2.13.0%2Flinux-chrome-114.0.0%2Fbrowser%3Arainbowkit-example-git-fork-tcadle434-feat-a86cc4-rainbowdotme.vercel.app&useOnCloseEvent=true' failed:

**Comment by DanielSinclair** [comment]
Have you supplied a valid WalletConnect API Key? Are you using a VPN or adblocker?
